### ISSUE TITLE
A blog post by me corrupts all the feeds of my followers and my blog

### ISSUE THREAD

**Issue by frnxstd** [steemit/condenser#2693]
Hello,

We are developing an application on Steem blockchain. The app accidentally posted through Steemit via Steem Connect v2 then everything related with my posts disappeared.

Here is my account: https://steemit.com/@ikidnapmyself
And it returns "Sorry! This page doesn't exist."

Here is the post that making trouble: https://steemd.com/dngo/@ikidnapmyself/my-awesome-title

Any follower of mine can not reach their feeds and my page too.

Just in case; everything work properly except the post. I am able to post, edit etc. And I tried to delete my post using Postman with comment_delete scope to broadcast the operation, it did not work.
```
{
    "error": "unauthorized_client",
    "error_description": "This access_token allow you to broadcast transaction only for the account @ikidnapmyself"
}
```
 I also with SteemJS GitHub example. There is a reply on the post so I can not delete it.
```
{
  "cause": {},
  "isOperational": true,
  "code": -32000,
  "data": {
    "code": 10,
    "name": "assert_exception",
    "message": "Assert Exception",
    "stack": [
      {
        "context": {
          "level": "error",
          "file": "steem_evaluator.cpp",
          "line": 467,
          "method": "do_apply",
          "hostname": "",
          "timestamp": "2018-03-25T02:12:27"
        },
        "format": "comment.children == 0: Cannot delete a comment with replies.",
        "data": {}
      },
```

Thanks in advance!

**Comment by roadscape** [comment]
Thanks for reporting this!

**Comment by Smittyvb** [comment]
This is similar to #2215. It appears that setting ```tags``` in ```json_metadata``` to anything other than an array of strings will cause this issue. Condenser should ignore invalid tags if it finds them in the ```tags``` array.

**Comment by frnxstd** [comment]
true, we have just realized that, our toArray() method returns an object. thanks for your help.

**Comment by kirkins** [comment]
@originated I noticed you self-assigned, did you have a chance to see my PR? https://github.com/steemit/condenser/pull/2705
